zyngui/zynthian_gui_save_preset.py

```diff
@@ -38,7 +38,7 @@ def save_preset(self):
 			self.save_preset_create_bank_name = None
 			self.save_preset_bank_info = None
 			self.layer.load_bank_list()
-			if not self.layer.bank_list or not self.layer.bank_list[0][0]:
```

Comment:
Same for this. it's a new feature, not related to Pianoteq API implementation, and it affects all engines.

Reply:
Possibly but the enhancement is added to facilitate functionality in pianoteq and is not previously implemented.